src/features.ts

```diff
@@ -1,4 +1,6 @@
 import { execSync } from 'node:child_process'
+import { pathToFileURL } from 'node:url'
```

Comment:
Tip: if you use `pathToFileURL` from `mlly`, it handles windows situation and returns string

Reply:
I see, was updating based on https://github.com/unjs/nitro/pull/732/files

Reply:
Updated with https://github.com/nuxt-hub/core/pull/317/commits/49f67b5b3965382755f7e74ab9fa5157d044b82f